adafruit_pixel_framebuf.py

```diff
@@ -38,16 +38,24 @@
 """
 
 # imports
+try:
+    from typing import Union
+
+    from adafruit_dotstar import DotStar
```

Comment:
I think we can just use this instead of importing from dotstar and neopixel
```suggestion
    from circuitpython_typing import FillBasedColorUnion
```